controllers/fenceagentsremediation_controller.go

```diff
@@ -218,6 +233,8 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 
 		r.Executor.Remove(far.GetUID())
 		r.Log.Info("FenceAgentsRemediation CR has completed to remediate the node", "Node Name", req.Name)
+		commonEvents.NormalEvent(r.Recorder, node, utils.EventReasonNodeRemediationCompleted, utils.EventMessageNodeRemediationCompleted)
```

Comment:
Isn't this a duplicated event?

Reply:
Duplication of which event? Can you please elaborate?

Reply:
at lines 236 and 237 it both uses 
```
commonEvents.NormalEvent(r.Recorder, node, utils.EventReasonNodeRemediationCompleted, utils.EventMessageNodeRemediationCompleted)
```

and

```
commonEvents.RemediationFinished(r.Recorder, far)
```

which, I believe, should result in similar information

Reply:
Oho you mean this duplication 😅 
Yes, it is indeed redundant. I would keep the second one which is common between Medik8s operators.

Reply:
Looking again it makes sense to have double events. 
One is for the node resource and one is for the far CR.